dotnet/src/SemanticKernel/SkillDefinition/NativeFunctionDelegate.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel.AI.TextCompletion;
+using Microsoft.SemanticKernel.Orchestration;
+
+namespace Microsoft.SemanticKernel.SkillDefinition;
+
+internal delegate Task<FunctionResult> NativeFunctionDelegate(
```

Comment:
I know this is existing -- but why do we need textCompletion and requestSettings here?

Reply:
I had the same question here :) It looks like we pass `null` as `ITextCompletion` parameter:
![image](https://github.com/microsoft/semantic-kernel/assets/13853051/90c5c9e9-3b29-4b45-b754-5f8d0951e566)

It looks like there was/is desire to keep similar signatures for Semantic and Native functions, but it can be definitely simplified in case of native function.

Reply:
Yeah, leave it for now (to avoid a lot more churn) and we'll fix up the remainder of ISKFunction in another PR. There are several semantic-only methods still in there.